Tribler/Test/Core/Category/test_category.py

```diff
@@ -18,9 +18,9 @@ def test_get_category_names(self):
         self.assertEquals(len(self.category.category_info), 10)
 
     def test_calculate_category_multi_file(self):
-        torrent_info = {b"info": {b"files": [{b"path": b"/my/path/video.avi", b"length": 1234}]},
+        torrent_info = {b"info": {b"files": [{b"path": [b"my", b"path", b"video.avi"], b"length": 1234}]},
```

Comment:
IIRC, in the metainfo returned by libtorrent, the `path` is not a dictionary consisting of an array of path components?

Reply:
I've checked this, and apparently `path` is a list of path components.

Reply:
Ah ok, good to know! 👍